jormungandr/src/blockchain/chain.rs

```diff
@@ -116,7 +117,13 @@ impl Blockchain {
 
                 let mut epoch = block_0.date().epoch;
                 let initial_leadership = Leadership::new(epoch, &state);
-                let mut leaderships = Leaderships::new(&block_0.header, initial_leadership);
+
+                let initial_gc_root = leaderships.add(
```

Comment:
This seems unused, why is this change done to the legacy blockchain code?

Reply:
because it was wrong the way it works. We never kept the initial gc_root. GC Run of the multiverse would have created a cascade of error. This is placeholder for when we do the switch to the new data structure.